ssl

```diff
@@ -72,59 +91,54 @@ $(ColorGreen './ssl domain.com')
 ;;
 *)
 	today=$(date +%F)
-	expires=$(echo | openssl s_client -servername $1 -connect $1:443 2>/dev/null | openssl x509 -noout -dates | grep 'notAfter' | sed 's/notAfter=//')
-
-	echo -ne "$(ColorRed '#') $(ColorGreen 'Certificate Chains:')"
-	echo ''
-	timeout 2 openssl s_client -quiet -showcerts -servername $1 -connect $1:443 
-	echo "$(ColorRed '----')"
+	expires=$(echo|openssl s_client -servername $1 -connect $1:443 2>/dev/null|openssl x509 -noout -dates|grep 'notAfter'|sed 's/notAfter=//')
 
-	echo -ne "$(ColorRed '#') $(ColorGreen 'Certificate Issued For:')"
-	echo ''
-	echo | openssl s_client -servername $1 -connect $1:443 2>/dev/null | openssl x509 -noout -subject | sed 's/subject=/Domain: /'
-	#echo | openssl s_client -servername www.$1 -connect www.$1:443 2>/dev/null | openssl x509 -noout -subject
+	echo -e "$(ColorRed '#') $(ColorGreen 'Leaf Certificate Issued For:')"
+	echo|openssl s_client -servername $1 -connect $1:443 2>/dev/null|openssl x509 -noout -subject|sed 's/subject=/Domain: /'
 	echo "$(ColorRed '----')"
 
-        echo -ne "$(ColorRed '#') $(ColorGreen 'Certificate Expires In:')"
-	echo ''
-
+   echo -e "$(ColorRed '#') $(ColorGreen 'Leaf Certificate Expires In:')"
 	# macos - format
 	#date -j -f "%b %d %T %Y %Z" "$expires" +'%s'
 	#date -j -f "%F" "$today" +'%s'
 	## echo $(( ( $(date -ud "$expires" +'%s') - $(date -ud "$today" +'%s') )/60/60/24 )) days
-	echo $(( ( $(date -j -f "%b %d %T %Y %Z" "$expires" +'%s') - $(date -j -f "%F" "$today" +'%s') )/60/60/24 )) days
+	echo "	$(( ( $(date -j -f "%b %d %T %Y %Z" "$expires" +'%s') - $(date -j -f "%F" "$today" +'%s') )/60/60/24 )) days"
```

Comment:
That looks great! Thank you for your contribution.

It looks like that on line 105 the `-j`  argument is throwing the following error:

```
date: invalid option -- 'j'
Try 'date --help' for more information.
date: invalid option -- 'j'
Try 'date --help' for more information.
ssl3: line 105: (  -  )/60/60/24 : syntax error: operand expected (error token is ")/60/60/24 ")
```

Reply:
Forgot to put in checks for OS version...ie macos/linux....am putting in checks for that soon.